strimzi-kafka-operator.yaml

```diff
@@ -0,0 +1,55 @@
+package:
+  name: strimzi-kafka-operator
+  version: 0.43.0
+  epoch: 0
+  description: Apache Kafka® running on Kubernetes
+  copyright:
+    - license: Apache-2.0
+
+environment:
+  contents:
+    packages:
+      - bash
+      - busybox
+      - ca-certificates-bundle
+      - coreutils
+      - curl
+      - docker # not gonna be needed but their script requires docker to be installed
+      - helm
+      - make
+      - maven
+      - openjdk-17
+      - openjdk-17-default-jvm
+      - wget
+      - yq
+      - zip
+
+pipeline:
+  - uses: git-checkout
+    with:
+      repository: https://github.com/strimzi/strimzi-kafka-operator
+      tag: ${{package.version}}
+      expected-commit: 23c0fb0726e1f212c6182eb251ff23697e72a353
+
+  - runs: |
+      export JAVA_HOME="/usr/lib/jvm/java-17-openjdk"
+      # needs shellcheck
+      scversion="stable"
+      wget -qO- "https://github.com/koalaman/shellcheck/releases/download/${scversion?}/shellcheck-${scversion?}.linux.${{build.arch}}.tar.xz" | tar -xJv
```

Comment:
I was going to say "Should it be separate package?" but then noticed that we don't have `cabal` or `haskell` packaged in Wolfi. 🤔

Reply:
Not crazy about this. Have we explored packaging it?

Reply:
Sadly cabal and haskell are not bootstrappable from source (many have tried in the haskell community but the original sources and compiler are lost). It would take a larger foundations effort to get those into extra-packages in a reasonably verifiable manner, which is beyond what we can do here.

Reply:
All that said, can this checkout use the git fetch pipeline? Or at least checkout a commit hash? Failing those, shasum verification of the tarball?

Reply:
@developer-guy see above, can we get the `fetch` pipeline implemented here?

Reply:
it looks a bit ugly but yeah I made it finally 🫠